daemon/execdriver/driver.go

```diff
@@ -24,10 +24,16 @@ var (
 	ErrDriverNotFound          = errors.New("The requested docker init has not been found")
 )
 
-// StartCallback defines a callback function.
+const (
```

Comment:
ha..just noticed (and fixed) that locally as I was attempting to convert to a struct :)